{
  "candidate": "CVE-2020-13281",
  "description": "For GitLab before 13.0.12, 13.1.6, 13.2.3 a denial of service exists in the project import feature",
  "priority": "medium",
  "cvss": {
    "nvd": {
      "vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
      "score": 6.5,
      "severity": "MEDIUM"
    }
  },
  "references": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-13281",
    "https://about.gitlab.com/releases/2020/08/05/gitlab-13-2-3-released/"
  ],
  "public_date": "2020-08-13T14:15:00Z",
  "packages": [
    {
      "name": "gitlab",
      "status": "needs-triage"
    }
  ]
}
